runtimes/eden/src/migrations.rs

```diff
@@ -0,0 +1,77 @@
+use crate::Runtime;
+use frame_support::{migration, parameter_types, traits::OnRuntimeUpgrade, weights::Weight, BoundedVec};
+use primitives::AccountId;
+
+#[cfg(feature = "try-runtime")]
+use frame_support::traits::OnRuntimeUpgradeHelpersExt;
+
+// MaxMembers is chosen based on what used to be the MaxMembers param for the pallet ValidaorsSet
+// We have intentionally used the same number for MaxInvulnerables for the pallet CollatorSelection
+// So the BoundedVec storage in these two pallets remain compatible.
+parameter_types! {
+	pub const MaxMembers: u32 = 50;
+}
+
+pub struct MoveValidatorsSetToInvulnerables;
+impl OnRuntimeUpgrade for MoveValidatorsSetToInvulnerables {
+	fn on_runtime_upgrade() -> Weight {
```

Comment:
Shall we configure the candidacy bond too?

Reply:
We can if we have a decision on how much it should be. We can also do that after the upgrade. What do you suggest?

Reply:
Can you set it to 3M NODL? I think that's what was floating around internally for now

Reply:
Done